src/components/movieListPage/MovieListPage.tsx

```diff
@@ -0,0 +1,209 @@
+import { useState, useEffect } from "react";
+import SearchForm from "../searchForm/SearchForm";
+import GenreSelect from "../genreSelect/GenreSelect";
+import MovieDetails from "../movieDetails/MovieDetails";
+import { type MovieFormData } from "../movieForm/MovieForm";
+import {
+  mapMovie,
+  MovieDialogType,
+  MovieSort,
+  type Movie,
+  type SortOption,
+} from "../../types/movie";
+import MoviesList from "../movieList/MovieList";
+import {
+  AVAILABLE_FORM_GENRES,
+  DEFAULT_MOVIE_GENRE,
+  MOVIE_GENRES,
+  MOVIES_API_URL,
+} from "../../constants/movie";
+import SortControl from "../sortControl/SortControl";
+import "./MovieListPage.css";
+import MovieDialog from "../movieDialog/MovieDialog";
+
+const MovieListPage = () => {
+  const [movies, setMovies] = useState<Movie[]>([]);
+  const [searchText, setSearchText] = useState<string>("");
```

Comment:
Something empty searches are allowed in the UI, so null could make more sense here.
```suggestion
  const [searchText, setSearchText] = useState<string | null>(null);
```